integration-tests/test_register.py

```diff
@@ -0,0 +1,47 @@
+import contextlib
+import pytest
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
this import && variables are not used

Reply:
fixed. I used the logger to see what is the current proxy setting.